internal/impl/redis/processor.go

```diff
@@ -59,11 +60,10 @@ performed for each message and the message contents are replaced with the result
 			Description("The time to wait before consecutive retry attempts.").
 			Default("500ms").
 			Advanced()).
-		LintRule(`
-root = if this.contains("operator") && this.contains("command") {
```

Comment:
I believe this lint rule wasn't functioning as expected before, since `contains()` only scans the values of the keys from the object, excluding subobjects (maybe we should add a `contains_rec()` method or update the docs to mention that `contains()` isn't recursive?).

I had to do `this.command != ""` because it has `Default("")`, so it always exists.